src/index.html

```diff
@@ -1,17 +1,211 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST"
+      onsubmit="onSubmit()">
+      <div class="block">
+        <fieldset>
+          <legend>Personal information</legend>
+          <div class="form-field">
+            <label for="surname-user">
+              Surname:
+            </label>
+            <input
+              type="text"
+              name="surname"
+              id="surname-user"
+              required
+            >
+          </div>
+          <div class="form-field">
+            <label for="name-user">
+              Name:
+            </label>
+            <input
+              type="text"
+              name="name"
+              id="name-user"
```

Comment:
```suggestion
              name="name"
              id="name"
```
Можна використовувати ті ж назви, що було більш зрозуміло

Reply:
Понял, Спасибо!)